config/config.go

```diff
@@ -153,6 +155,9 @@ type QueryResponse struct {
 	StartTLS bool   `yaml:"starttls,omitempty"`
 }
 
+type GRPCHEALTHV1Probe struct { 
```

Comment:
In Go, names are generally camel case (MixedCase) with initialisms/acronyms capitalised and the rest MixedCase, so this would be `GRPCHealthV1Probe` - that's a little more readable too.

ref: https://talks.golang.org/2014/names.slide#5

Reply:
Thanks @camh-anz. New to golang but will eventually work out what idiomatic golang looks like.